qemu/tests/sgx_basic.py

```diff
@@ -0,0 +1,44 @@
+from virttest import error_context
+from virttest.utils_misc import verify_dmesg
+
+from provider.sgx import SGXHostCapability
+from provider.sgx import SGXChecker
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Qemu sgx basic test:
+    1. Check host sgx capability
+    2. Boot sgx VM
+    3. Check sgx qmp cmd
+    4. Verify sgx flags and epc size
```

Comment:
Here I suggest we specify what should be verified in the steps,e.g.
1. Validate sgx is supported by the host, and get the total epc size

Reply:
Hi @zhencliu, I have a different understanding of it. I think this is only an overall description of the case test steps, what to verify of each qmp cmd is too detailed and this is well explained in the functions. I tend to keep it simple like check qmp and verify qmp, like 
    1. Check host sgx capability
    2. Boot sgx VM
    3. Verify sgx enabled in guest
    3. Check sgx qmp cmd
    4. Verify qmp cmd in host and guest sgx capability

Reply:
Agreed. No need more details. 

It may be different for people on what the details are, e.g. it looks to me that the two qmp cmds are not part of details, because one is for host and the other is for guest(I have no idea without your explanation), but the checking items in the outputs of the cmds should be details, which are not needed here.

My concern is
1. this is open source code, so we can make the steps clear to help people outside understand  it.
2. even for people in our own team, one day, if another maintainer review this code, he/she could easily understand it with these steps, without checking the original manual case

Finally, it's OK for me with the current, but as we said, we can consider this part in new case dev in future, it's also important.

Thanks.